cmd/tailscaled/tailscaled.go

```diff
@@ -806,6 +811,35 @@ func beChild(args []string) error {
 	return f(args[1:])
 }
 
+var serveTailfsFunc = serveTailfs
```

Comment:
It's used in the below, and I can't take the address of `serveTailfs` directly.

```
var subCommands = map[string]*func([]string) error{
	"install-system-daemon":   &installSystemDaemon,
	"uninstall-system-daemon": &uninstallSystemDaemon,
	"debug":                   &debugModeFunc,
	"be-child":                &beChildFunc,
	"serve-tailfs":            &serveTailfsFunc,
}
```

Reply:
Yeah, I was assuming we would make the type not use a `*`, since there's no real need for it. A function is already a reference type, so it defaults to nil.